teams/tiflash/membership.json

```
@@ -0,0 +1,44 @@
+{
+    "_comment": "Items of lists in this file are in alphabetical order. Please keep it on modification.",
+
+    "description": "TiFlash team is responsible for Analytical Processing (AP) part in the Hybrid Transactional/Analytical Processing (HTAP) architecture of TiDB.",
+
+    "maintainers": [
+        "flowbehappy",
+        "ilovesoup",
+        "zanmato1984"
+    ],
+
+    "committers": [
+        "CalvinNeo",
+        "JaySon-Huang",
+        "Lloyd-Pottiger",
+        "SeaRise",
+        "XuHuaiyu",
+        "bestwoody",
+        "birdstorm",
+        "fuzhe1989",
+        "fzhedu",
+        "gengliqi",
+        "guo-shaoge",
+        "hehechen",
+        "ichn-hu",
+        "jiaqizho",
+        "lidezhu",
+        "littlefall",
+        "mengxin9014",
+        "solotzg",
+        "windtalker",
+        "wshwsh12",
+        "yibin87"
```

Suggest adding @JinheLin as committers

```suggestion
        "yibin87",
        "JinheLin"
```